interactive_ai/supported_models/geti_supported_models/parser.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2022-2025 Intel Corporation
+# LIMITED EDGE SOFTWARE DISTRIBUTION LICENSE
+
+import hiyapyco
+from importlib import resources
+
+from geti_supported_models import manifests
+from geti_supported_models.model_manifest import ModelManifest
```

Comment:
```suggestion
from .model_manifest import ModelManifest
```
You can use relative imports within the package